test/embeds-test.js

```diff
@@ -116,8 +118,10 @@ test('InstagramEmbed - onResize', t => {
   const el = document.body.appendChild(document.createElement('div'));
   const expectedHeight = 754;
   const onResize = ({height}) => {
-    t.equal(height, expectedHeight);
-    t.end();
+    setTimeout(() => {
+      t.equal(height, expectedHeight);
+      t.end();
+    }, 0);
```

Comment:
perhaps comment or so about why we need this? Can easily be removed since it looks unnecessary but is needed...